pandas/core/frame.py

```diff
@@ -2523,16 +2523,25 @@ def to_html(
     @Substitution(
         klass="DataFrame",
         type_sub=" and columns",
-        max_cols_sub=(
-            """max_cols : int, optional
+        max_cols_sub=dedent(
```

Comment:
can you do a followup where you switch to using `@doc` before the Series.info change